src/Page/PageCanvas.jsx

```diff
@@ -132,7 +151,13 @@ export class PageCanvasInternal extends PureComponent {
     const cancellable = page.render(renderContext);
     this.renderer = cancellable;
 
-    return cancellable.promise.then(this.onRenderSuccess).catch(this.onRenderError);
+    return (
+      cancellable.promise
+        // Show the canvas on success and on errors
+        .then(this.onRenderSuccess)
+        .catch(this.onRenderError)
+        .finally(this.showCanvas)
```

Comment:
In our fork we actually implemented the following and can confirm it works. However I noticed it failed a test so I changed it to `finally`, which I believe should have the same effect barring a little different timing.
```js
.then(this.showCanvas, this.showCanvas)
.then(this.onRenderSuccess)
.catch(this.onRenderError)